### DISCUSSION TITLE
Compiling SketchyBar on Catalina

### DISCUSSION THREAD

**Discussion by dinulashenal** [FelixKratz/SketchyBar#146] [Q&A] [answered]
Are you can give me the Sketchy Bar Dmg File ?
I want the Sketchy Bar Dmg File because I can't install it using homebrew.I install Sketchy Bar using homebrew then comes problems "1st one is  Sketchy Bar formula is not installed and 2nd one is Cmake problem.

**Comment by fruitless** [comment]
You can compile it by yourself. Simply download source code, cd to **Sketchy-master** folder, then type in terminal:
```
mkdir ./bin
clang src/manifest.m -std=c99 -Wall -DNDEBUG -Ofast -fvisibility=hidden -F/System/Library/PrivateFrameworks -framework Carbon -framework Cocoa -framework SkyLight -o bin/sketchybar
./bin/sketchybar
```
If its working, move binary file:
`mv ./bin/sketchybar /usr/local/bin/sketchybar`

And add this lines to the end of yabairc for more convenient launch via brew services:
```
killall sketchybar &> /dev/null
sketchybar &> /dev/null &
```

**Reply by dinulashenal** [reply]
It's not Working...

**Comment by FelixKratz** [comment]
When you have a fresh clone of the repo, simply run `make x86`. This will generate the sketchybar_x86 binary without the arm64 instructions.

**Comment by FelixKratz** [comment, marked as answer]
I think installing on Catalina via Brew should now work as expected.
